lib/double_entry/reporting/aggregate_array.rb

```diff
@@ -50,8 +53,9 @@ def retrieve_aggregates
         scope = LineAggregate.
                 where(:function => function).
                 where(:range_type => 'normal').
-                where(:account => account.to_s).
-                where(:code => code.to_s).
+                where(:account => account.try(:to_s)).
+                where(:code => code.try(:to_s)).
+                where(:partner_account => partner_account.try(:to_s)).
```

Comment:
We pushed them :)!  https://github.com/envato/double_entry/blob/536d6faa9bff6b65b8c92e782844f62869afaa34/spec/double_entry/reporting/aggregate_array_spec.rb